testframework/build.gradle

```diff
@@ -10,11 +10,12 @@ java.withSourcesJar()
 apply plugin : net.neoforged.minecraftdependencies.MinecraftDependenciesPlugin
 
 dependencies {
-    // TODO: is this leaking in the POM? (most likely yes)
-    // TODO: does this need to be changed back to runtimeDependencies?
-    // TODO: should use attributes to resolve the right variant instead of hardcoding
-    compileOnly project(path: ':neoforge', configuration: 'apiElements')
-    runtimeOnly project(path: ':neoforge', configuration: 'runtimeElements')
+    compileOnly(project(path: ':neoforge')) {
```

Comment:
As elsewhere, this logic seems wrong. In fact, you should just be able to leave it at `compileOnly(project(path: ':neoforge'))` -- there's no runtime selection happening, so there's no risk of grabbing runtimeElements. Well, unless you do this, where it'll explicitly grab that when it _should_ grab only the apiElements.

Reply:
Good point, nothing to do here.